CHANGELOG.md

```diff
@@ -5,7 +5,11 @@
     * History now shows what was typed for macros and not the resolved value by default. This is consistent with
     the behavior of aliases. Use the `expanded` or `verbose` arguments to `history` to see the resolved value for
     the macro.
+    * Fixed parsing issue in case where output redirection appears before a pipe. In that case, the pipe was given
```

Comment:
This PR contains a really nice set of enhancements and bug fixes that make `cmd2` applications behave significantly closer to how a "normal" shell behaves.  Nice work!